{
  return new TestCase("warning"){
    protected void runTest(){
      fail(message);
    }
  }
;
}
